enterprise_access/apps/api/tasks.py

```diff
@@ -127,6 +130,50 @@ def send_notification_email_for_request(
     )
 
 
+@shared_task(base=LoggedTaskWithRetry)
+def send_cancel_email_for_pending_assignment(cancelled_assignment):
+    """
+    Send email via braze for cancelling pending assignment
+
+    Args:
+        cancelled_assignment: (string) the subsidy request uuid
+    """
+    learner_content_assignment_action = LearnerContentAssignmentAction(
+        uuid=cancelled_assignment.uuid, assignment=cancelled_assignment, action_type=AssignmentActions.CANCELLED
+    )
+
+    if braze_trigger_properties is None:
+        braze_trigger_properties = {}
+
+    braze_client_instance = BrazeApiClient()
+    lms_client = LmsApiClient()
+    enterprise_customer_uuid = learner_content_assignment_action.assignment_configuration.enterprise_customer_uuid
```

Comment:
The `assignment_configuration` is a member of the `LearnerContentAssignment` model, not the action model. You could instead get to the customer uuid by doing this:
```python
enterprise_customer_uuid = cancelled_assignment.assignment_configuration.enterprise_customer_uuid
```